inference-engine/src/mkldnn_plugin/nodes/mkldnn_topk_node.cpp

```diff
@@ -475,4 +1836,8 @@ inline int MKLDNNTopKNode::count(SizeVector dims, size_t start_ind) {
     return count(dims, start_ind, dims.size());
```

Comment:
Both count methods can be made static.
After this a lot of top* methods can be made const.

Reply:
Good point! Applied. Thanks Egor!